### PR TITLE
27 frontend   add notes

### LINKED ISSUE #27: FRONTEND - Add notes
Look figma. Want a page where user can make a note.

### PR DESCRIPTION
**Pull request by Valde1998**
👍

### CODE REVIEW

File: src/AddNote.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
```
Comment (DoctorHeyde) [inline src/AddNote.jsx:4]: Would like style components at the bottom (this is just preference).

File: src/AddNote.jsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  border-radius: 5px;
+  border: 1px solid #080808;
+  width: 100%;
+  height: 200px;
+`;
+
+function AddNote() {
+  const [title, setTitle] = useState('');
+  const [note, setNote] = useState('');
+  const [topic, setTopic] = useState('');
```
Comment (DoctorHeyde) [inline src/AddNote.jsx:63]: Change "topic" to "category"

File: src/AddNote.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  const handleTitleChange = (event) => {
+    setTitle(event.target.value);
+  };
+
+  const handleNoteChange = (event) => {
+    setNote(event.target.value);
+  };
+
+  const handleTopicChange = (event) => {
+    setTopic(event.target.value);
```
Comment (DoctorHeyde) [inline src/AddNote.jsx:74]: topic

File: src/AddNote.jsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log('Titel: ', title);
+    console.log('Note: ', note);
+    console.log('Emne: ', topic);
+    setTitle('');
+    setNote('');
+    setTopic('');
```
Comment (DoctorHeyde) [inline src/AddNote.jsx:84]: topic

File: src/AddNote.jsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+  const handleTopicChange = (event) => {
+    setTopic(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log('Titel: ', title);
+    console.log('Note: ', note);
+    console.log('Emne: ', topic);
```
Comment (DoctorHeyde) [inline src/AddNote.jsx:81]: English not Danish

**Review by TRossen89 — APPROVED** [review]

**Pull request merged**